go.mod

```diff
@@ -0,0 +1,34 @@
+module github.com/snapcore/snapd
+
+go 1.13
+
+require (
+	github.com/boltdb/bolt v1.3.1 // indirect
+	github.com/canonical/go-sp800.90a-drbg v0.0.0-20210314144037-6eeb1040d6c3 // indirect
+	github.com/canonical/go-tpm2 v0.0.0-20210314160024-32171bd353b1
+	github.com/canonical/tcglog-parser v0.0.0-20200908165021-12a3a7bcf5a1 // indirect
+	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7
+	github.com/godbus/dbus v0.0.0-20190726142602-4481cbc300e2
+	github.com/gorilla/mux v1.7.4-0.20190701202633-d83b6ffe499a
+	github.com/gvalkov/golang-evdev v0.0.0-20191114124502-287e62b94bcb
```

Comment:
I'm surprised if this works, it didn't work for me when I tried building with upstream because the tests in that package are broken

Reply:
it's not working yet :/

Reply:
It seems it is working afterall, no idea why it works here but did not for you.